server/src-lib/Hasura/GraphQL/Context.hs

```diff
@@ -86,31 +87,31 @@ traverseAction f = \case
   RFAction x -> RFAction <$> f x
   RFRaw x -> pure $ RFRaw x
 
-data QueryDB v
-  = QDBSimple      (RQL.AnnSimpleSelG       v)
-  | QDBPrimaryKey  (RQL.AnnSimpleSelG       v)
-  | QDBAggregation (RQL.AnnAggregateSelectG v)
-  | QDBConnection  (RQL.ConnectionSelect    v)
+data QueryDB b v
+  = QDBSimple      (RQL.AnnSimpleSelG       b v)
+  | QDBPrimaryKey  (RQL.AnnSimpleSelG       b v)
+  | QDBAggregation (RQL.AnnAggregateSelectG b v)
+  | QDBConnection  (RQL.ConnectionSelect    b v)
 
-data ActionQuery v
-  = AQQuery !(RQL.AnnActionExecution v)
-  | AQAsync !(RQL.AnnActionAsyncQuery v)
+data ActionQuery (b :: Backend) v
```

Comment:
Backend parameter is not applicable to actions.

Reply:
In which sense is the parameter less applicable here than it is for `QueryDB`?

(Note that 6 lines below the line you quoted, indeed we fix the parameter to be `'Postgres`. At a later stage we can generalize this code to allow actions to work for different backends.)

Reply:
Ah, I see what you mean.